lib/tldr.js

```diff
@@ -23,34 +23,33 @@ class Tldr {
 
   list(singleColumn) {
     let os = platform.getPreferredPlatformFolder(this.config);
-    index.commandsFor(os)
+    return index.commandsFor(os)
```

Comment:
This could potentially be refactored to use async / await to reduce visual complexity

Reply:
Sure. I didn't see async/await in there so I didn't want to introduce it, but I do prefer using it so if you'd prefer that syntax I can update it.

Reply:
Up to you @mearns. Since you have already spent your time on it, I don't want to redo it again.

Reply:
Looks like the projects current configuration doesn't support async/await syntax; it comes up with linting errors. I tried a few changes to address it, including updating to eslint version 7 and setting`parserOptions.ecmaVersion` to 7, but neither of these worked.

More importantly, these keywords [aren't supported before Node v7.10](https://node.green/#ES2017-features-async-functions), so to add them would require dropping support for older versions of Node (currently specified in package.json:engines as `>=6.12.0`), or else adding a transpiler.

In light of this, I propose that adding async/await is out of scope for this change and is a decision that the project owners need to make. If the decision is made to add them, I'd be happy to assist with that, but I propose it should be in a separate change set (a dedicated PR and release).

Reply:
Let's go with the existing code for now.